src/commands/tag/list.ts

```diff
@@ -21,7 +21,7 @@ import * as cf from '../../utils/common-flags';
 import { getBalenaSdk, getVisuals, stripIndent } from '../../utils/lazy';
 import { applicationIdInfo } from '../../utils/messages';
 
-export default class TagsCmd extends Command {
+export default class TagListCmd extends Command {
```

Comment:
No aliases & deprecateAliases in this command?

Reply:
Nice catch